guides/getting-started.md

```diff
@@ -1,196 +1,99 @@
+### New to Angular ?
 
-You can use either the npm or yarn command-line tool to install packages. Use whichever is appropriate for your project in the examples below.
+If you are new to Angular or getting started with a new Angular application, see [Angular's full Getting Started Guide](https://angular.io/start) and [Setting up your environment](https://angular.io/guide/setup-local).
 
-#### NPM
-```bash
-npm install --save @angular/material @angular/cdk @angular/animations
-```
-#### Yarn
-```bash
-yarn add @angular/material @angular/cdk @angular/animations
-```
-
-
-#### Alternative 1: Snapshot Build
+For existing applications, follow the steps below to begin using Angular Material:
 
-A snapshot build with the latest changes from master is also available. Note that this snapshot
-build should not be considered stable and may break between releases.
 
-#### NPM
-```bash
-npm install --save angular/material2-builds angular/cdk-builds angular/animations-builds
-```
-
-#### Yarn
-```bash
-yarn add angular/material2-builds angular/cdk-builds angular/animations-builds
-```
-#### Alternative 2: Angular Devkit 6+
+### Step 1: Install Angular Material npm packages
 
-Using the Angular CLI `ng add` command will update your Angular project with the correct dependencies, perform configuration changes and execute initialization code.
+Use the Angular CLI's install [schematic](https://material.angular.io/guide/schematics) to set up your Angular Material project by running the following command:
 
 ```bash
 ng add @angular/material
 ```
 
-### Step 2: Configure animations
+The `ng add` command will install Angular Material, the [Component Dev Kit (CDK)](https://material.angular.io/cdk/categories), [Angular Animations](https://angular.io/guide/animations) and ask you the following questions to determine which features to include:
 
-Once the animations package is installed, import `BrowserAnimationsModule` into your application to enable animations support.
+1. Choose a prebuilt theme name, or "custom" for a custom theme: 
 
-```ts
-import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+   You can choose from [prebuilt material design themes](https://material.angular.io/guide/theming#using-a-pre-built-theme) or set up an extensible [custom theme](https://material.angular.io/guide/theming#defining-a-custom-theme).
 
-@NgModule({
-  ...
-  imports: [BrowserAnimationsModule],
-  ...
-})
-export class PizzaPartyAppModule { }
-```
+2. Set up HammerJS for gesture recognition:
 
-Alternatively, you can disable animations by importing `NoopAnimationsModule`.
+   [HammerJS](http://hammerjs.github.io/) provides gesture recognition capabilities required by some components (`mat-slide-toggle`, `mat-slider`, `matToolTip`).
 
-```ts
-import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+   Please note, if you choose not to install HammerJS it can be installed later via npm using the following command:
```

Comment:
I would rather avoid putting this in the main content of the guide, instead using an appendix at the end. It's also somewhat moot since we're removing the dependency on hammer very soon. 

I would also skip the install instructions (_especially_ the different `npm` and `yarn` blocks) since it adds a lot of noise for something 99% of people reading this will know how to do already.

Reply:
Done. Please review.